subsys/bluetooth/host/audio/bap.c

```diff
@@ -138,8 +138,13 @@ static int bap_reconfig(struct bt_audio_chan *chan,
 		tmp->codec = codec;
 	}
 
-	/* Unbind if there is an existing QoS */
-	bt_audio_chan_unbind(chan);
+	/* Terminate CIG if there is an existing QoS,
```

Comment:
I may need some education here: The following code seems to terminate the CIG unconditionally.  Where is the "if there is an existing QoS"?

Reply:
Added a check